### PR TITLE
Working on new theme (#2973)

### PR DESCRIPTION
**Pull request by justinsoltesz**
This includes all of the work on the 1.9 theme to date. Progress is being tracked in #2973 

This also fixes the CSS references so that the theme can actually be used. **The references are temporary until the new theme system is incorporated.** Includes external references to Google Fonts and FontAwesome as well as using the CSS in [mybb/1.9-ui-framework](https://github.com/mybb/1.9-ui-framework) for the overall theme.

### CODE REVIEW

**Comment by euantorano** [comment]
A note for a backend/frontend change: https://community.mybb.com/thread-215211-post-1317265.html#pid1317265

**Comment by justinsoltesz** [comment]
Yep, get that change included soon.

File: private.php [hunk 1]
```diff
@@ -903,7 +903,11 @@
         $db->update_query('privatemessages', $updatearray, "pmid='{$pmid}'");
 
         // Update the unread count - it has now changed.
-        update_pm_count($mybb->user['uid'], 6);
+        $pmcount = update_pm_count($mybb->user['uid'], 6);
+
+		if(is_array($pmcount) && isset($pmcount['unreadpms'])) {
```
Comment (euantorano) [inline private.php]: Indentaion here seems a bit messed up, should be:

```suggestion
    if (is_array($pmcount) && isset($pmcount['unreadpms'])) {
        $mybb->user['pms_unread'] = $pmcount['unreadpms'];
    }
```

File: private.php [hunk 2]
```diff
@@ -973,7 +977,9 @@
 
     if (count($pm['bcc_recipients']) > 0) {
         $pm['bcc_form_val'] = implode(',', $pm['bcc_form_val']);
-    }
+    } else {
+		$pm['bcc_form_val'] = '';
```
Comment (euantorano) [inline private.php]: Again, indentation:

```suggestion
        $pm['bcc_form_val'] = '';
    }
```

**Comment by euantorano** [comment]
As discussed yesterday, I'm merging this PR now before I rebase the `develop/1.9` branch since we released `1.8.20` yesterday.

**Pull request merged**